src/IssueViz/IssueVisualizationControl/ViewModels/Commands/NavigateToRuleDescriptionCommand.cs

```diff
@@ -34,13 +34,18 @@ internal interface INavigateToRuleDescriptionCommand : ICommand
     internal class NavigateToRuleDescriptionCommand : DelegateCommand, INavigateToRuleDescriptionCommand
     {
         [ImportingConstructor]
-        public NavigateToRuleDescriptionCommand(IShowInBrowserService showInBrowserService)
+        public NavigateToRuleDescriptionCommand(IEducation educationService)
             : base(parameter =>
                 {
-                    var ruleKey = (string) parameter;
-                    showInBrowserService.ShowRuleDescription(ruleKey);
+                    var fullRuleKey = (string) parameter;
+                    if (SonarCompositeRuleId.TryParse(fullRuleKey, out var ruleId))
+                    {
+                        educationService.ShowRuleHelp(ruleId);
```

Comment:
Rather than calling `IShowInBrowserService`, the command calls `IEducation` instead.

`IEducation` is responsible for deciding whether to show the help in the IDE or the browser.